src/server/endpoints/covidcast.py

```diff
@@ -122,16 +122,61 @@ def guess_index_to_use(time: List[TimePair], geo: List[GeoPair], issues: Optiona
     return None
 
 
+# TODO: Write an actual smoother arg parser.
+def parse_smoother_args():
+    return {"window_length": 7, "pad_length": 6, "pad_fill_value": None, "nans_fill_value": 0.}
+
+
+def _extract_first_entry(iter: Iterable[Tuple[Any, Any]]) -> Iterable[Any]:
+    for x, _ in iter:
+        yield x
+
+
+def _get_transformed_row_generator(
+    parsed_rows: Iterable[Dict],
+    iterator_buffer: Generator[Iterable[Tuple[Dict, Union[Tuple[str, str], str]]], None, None],
+    is_compatibility: bool,
+    original_source: Optional[str] = None,
+    smoother_args: Dict = {},
```

Comment:
optional args with dict/lists shouldn't be used in my knowledge since it won't create new instances